lib/capybara/spec/session/check_spec.rb

```diff
@@ -189,7 +189,7 @@
           expect(end_time - start_time).to be < 10
         end
 
-        it 'should check via the label if input is moved off the left edge of the page' do
+        it 'should check via the label if input is visible but blocked by another element' do
```

Comment:
This test is actually relevant for non-selenium drivers - please don't change - add a new test if needed/wanted

Reply:
Can do. Do you have a preference for the next supercar brand to add? Bugatti, perhaps?

Reply:
Bugatti is definitely acceptable :)